tests/models/roberta_prelayernorm/test_modeling_flax_roberta_prelayernorm.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+
+from transformers import RobertaPreLayerNormConfig, is_flax_available
+from transformers.testing_utils import require_flax, slow
+
+from ...test_modeling_flax_common import FlaxModelTesterMixin, floats_tensor, ids_tensor, random_attention_mask
+
+
+if is_flax_available():
+    from transformers.models.roberta_prelayernorm.modeling_flax_roberta_prelayernorm import (
+        FlaxRobertaPreLayerNormForCausalLM,
+        FlaxRobertaPreLayerNormForMaskedLM,
+        FlaxRobertaPreLayerNormForMultipleChoice,
+        FlaxRobertaPreLayerNormForQuestionAnswering,
+        FlaxRobertaPreLayerNormForSequenceClassification,
+        FlaxRobertaPreLayerNormForTokenClassification,
+        FlaxRobertaPreLayerNormModel,
+    )
+
+
+class FlaxRobertaPreLayerNormModelTester(unittest.TestCase):
```

Comment:
Think we can use copied from here, would help to know what was actually added or not